website/content/userguide/permanent-data.markdown

```diff
@@ -0,0 +1,120 @@
+---
+title: "Access permanent data"
+menuTitle: "Access permanent data"
+weight: 49
+---
+
+## Create the filesystem
+
+gokrazy provides a handy way to create the permanent data filesystem on the forth partition of your gokrazy installation.
+The [gokrazy/mkfs](https://github.com/gokrazy/mkfs) program will do all the required steps to create the filesystem.
+To actually access the permanent data partition from your own program, gokrazy will mount the partition under `/perm` directory during the startup.
+
+You can either include this program in your `gokr-packer` command line:
+
+```
+gokr-packer \
+  -overwrite=/dev/sdx \
+  -serial_console=disabled \
+  github.com/gokrazy/fbstatus \
+  github.com/gokrazy/hello \
+  github.com/gokrazy/serial-busybox \
+  github.com/gokrazy/mkfs
+```
+
+…or, if you want to run it only once without otherwise including it in your
+installation, you can use `gok run`:
+
+```
+git clone https://github.com/gokrazy/mkfs
+cd mkfs
+gok run -i bakery
+```
+
+```bash
+sudo mkfs.ext4 /dev/sdc
+```
+
+## Permanent data in action
+
+The Go program demonstrates the functionality of the permanent data mount:
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+)
+
+const config = "/perm/config.json"
+
+type Config struct {
+	FilesToCreate int `json:"files_to_create"`
+}
+
+func main() {
+	_, err := os.Stat(config)
+	if err != nil {
+		fsErr := &fs.ErrNotExist
+		if !errors.As(err, fsErr) {
+			panic(err)
+		}
+
+		content, err := json.Marshal(&Config{FilesToCreate: 4})
+		if err != nil {
+			panic(err)
+		}
+
+		if err = ioutil.WriteFile(config, content, 0600); err != nil {
+			panic(err)
+		}
+	}
+
+	content, err := ioutil.ReadFile(config)
+	if err != nil {
+		panic(err)
+	}
+
+	var c Config
+	if err := json.Unmarshal(content, &c); err != nil {
+		panic(err)
+	}
+
+	for i := 0; i < c.FilesToCreate; i++ {
+		if err := ioutil.WriteFile(fmt.Sprintf("/perm/%d.txt", i), []byte("gokrazy rocks"), 0600); err != nil {
+			panic(err)
+		}
+	}
+}
+
+```
+
+## Manual inspection/modification
+
+You can inspect and modify the data on the permanent data filesystem manually.
+This can be handy if you need e.g. to pass a configuration file for your application.
+To access the filesystem put the SD card into your PC/notebook. Your OS will likely mount the filesystem automatically.
+
+gokrazy will access your files as `root`, which means you will have to set proper permissions before placing data on the filesystem.
+Note that the `chown` utility requires `root` privileges in order to set the owner to `root:root`.
+In Unix you can use `sudo` to gain `root` privileges, also known as the `break glass` process.
+With this process a user is capable to gain higher administrative privileges which is limited by a password time duration.
+After the duration is exceeded, the user has to re-authenticate again with his password.
```

Comment:
Probably easier to link to https://en.wikipedia.org/wiki/Sudo for people who are unfamiliar with sudo :)